src/connectors/__test__/stripe.test.ts

```diff
@@ -0,0 +1,71 @@
+import Stripe from 'stripe'
+
+import { PAYMENT_CURRENCY } from 'common/enums'
+
+import { stripe } from '../stripe'
+
+const testUser1 = {
+  id: '1',
+  email: 'test1@matters.news',
+}
+const testUser2 = {
+  id: '2',
+  email: 'test2@matters.news',
+}
+
+// const stripeAPI = (stripe as any).stripeAPI as Stripe
+let customer: Stripe.Customer
+
+test('create customer', async () => {
+  const _customer = await stripe.createCustomer({ user: testUser1 })
```

Comment:
Is it run on [Stripe mock](https://stripe.com/docs/automated-testing)?

Reply:
run on  https://github.com/stripe/stripe-mock

Reply:
https://github.com/thematters/matters-server/pull/3449/files#diff-dcd9bbd1ca93eef82a91d6b61ddfbf6502e4573161f62786d6ea188edeb1080bR41